DOCS/man/vo.rst

```diff
@@ -488,10 +488,19 @@ Available video output drivers are:
         Mode ID to use (resolution and frame rate).
         (default: 0)
 
-    ``--drm-overlay=<number>``
-        Select the DRM overlay index to use.
-        Overlay index is zero based, and related to crtc.
-        (default: 0)
+    ``--drm-osd-plane-id=<number>``
```

Comment:
Move the documentation of these options from this commit, to commit 540fec5b6379ea739842637c0ba9dbd472843c5f when they are introduced in the code.
Like this the documentation will have been introduced before the options.